examples/cliffordexamples/base-tableau.tabl

```diff
@@ -0,0 +1,3 @@
+2|1|2|3|4|R|
+1|1|1|0|0|0|
+2|0|0|1|1|0|
```

Comment:
Is that format in some way established? It's not really self explanatory to me. I would have assumed the most simple format to be a binary table/matrix representing the stabilizer tableau.

Reply:
Not established, but I choose it that way to offer some tradeoff between ease of use in code and readability. Its basically a Binary matrix with row/col numbers and the number of qubits in the first position

Reply:
Ill change the indexes to start from 0, but for the rest, I like that it is more readable that way (also possibly easier to convert into a csv with the separators)